src/components/Post.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import User from './User';
+
+export default class Post extends Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components